contributing/how_to_contribute.rst

```diff
@@ -0,0 +1,112 @@
+On this page we want to showcase the various ways you as users can participate - to help you find the right starting place with the skillset you have.
+Because contrary to popular opinion, we need more than just programmers on the project!
+
+
+Fundraising
+-----------
+
+- **Donate**
+
+  We created the non-profit `Godot Foundation <https://godot.foundation/>`_ to be able to support the Godot Engine in both matters of finance and administration. 
+  In action, this means we can hire people to work part-time or full-time on the project. 
+  These jobs include engine development as well as related tasks like code reviews, production management, community & marketing efforts, and more.
+
+  With as little as 5 EUR per month, you can help us keep going strong. 
+  Currently, we are intending to hire more core developers, as to cover more ground with full-time specialists that supplement and guide volunteer work.
+
+  `Join the Development Fund <fund.godotengine.org>`_
+
+- **Donation Drives**
+  Think about your followers on social media, or other communities you are active in.
+  Use that reach to remind your social environment that even small contributions can make a difference, especially when done by a great number of people at the same time. 
+
+  Are you a content creator? Consider adding a link to the `Godot Development Fund <fund.godotengine.org>`_ to your descriptions. 
+  If you do live streams, perhaps think about organizing a stream with donation incentives.
+
+.. - **Buy Official Merch**
+
+- **Publish Godot Games.**
+  You heard right, simply publishing a game #MadeWithGodot can positively impact the well-being of this project. 
+  Your personal success elevates the engine to a viable alternative for other developers, growing the community further. 
+  Additionally, it opens the doors for us to approach industry contacts about possible cooperations.
+
+
+Technical contributions
+-----------------------
+
+- **Report bugs & other issues**
+  As active users of the engine, you are better equipped to identify bugs and other issues than anyone else.
+  To let us know about your findings, fill out `this bug report form <https://github.com/godotengine/godot/issues/new>`_ on our GitHub.
+  Make sure to include as much information as possible to ensure these issues can easily be reproduced by others.
+
+  If you are interested in helping fix those problems you discovered, you may also join the `bugsquad <https://chat.godotengine.org/channel/bugsquad>`_!
+
+- **Test Development Versions**
+  While it is recommended to use the stable releases for your projects, you can help us test dev releases, betas, and release candidates
+  by opening a copy of your project in them and checking what problems this introduces or maybe even solves. 
+  Make sure to have a backup ready, since this can produce irreversible changes.
+
+  Find recent development versions `here <https://godotengine.org/download/preview/>`_
```

Comment:
`how_to_contribute.rst:4:Duplicate explicit target name: "here".`
one duplicate link remaining.


```suggestion
  You can find our recent development versions on our `godot preview builds <https://godotengine.org/download/preview/>`_ download page.
```

or at `Community-translations are hosted here` (line 91)

(yea I'm not good at writing..)
Theoretical, link names should be able to stand on their own, but I don't think this is normally considered in the docs.

Reply:
Adapted all links now, thank you!